app/main/views/users.py

```diff
@@ -63,6 +63,12 @@ def list_users():
             links={}
         )
 
+    only_buyers = request.args.get('only_buyers')
```

Comment:
Also agree a role filter makes sense, we have filters like this elsewhere where a comma-separated list of values we want can be passed in (eg for services on frameworks I think).